project-words.txt

```diff
@@ -758,4 +758,5 @@ PEPE
 LDO
 KNC
 HAPI
-DVF
\ No newline at end of file
+DVF
+dogecoin
```

Comment:
Can we keep these alphabetized btw? I guess it doesn't super matter but it's a nit LOL

Reply:
sure lol

Reply:
done @emilyJLin95